<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderSettingsWidget</class>
 <widget class="QWidget" name="RenderSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="widthLabel">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="widthSpinBox">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>8192</number>
        </property>
        <property name="singleStep">
         <number>360</number>
        </property>
        <property name="value">
         <number>1920</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="heightLabel">
        <property name="text">
         <string>Height </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="heightSpinBox">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>8192</number>
        </property>
        <property name="singleStep">
         <number>360</number>
        </property>
        <property name="value">
         <number>1080</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="samplesLabel">
        <property name="text">
         <string>Samples</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="SquareSpinBox" name="samplesSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20000</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="renderModeLabel">
        <property name="text">
         <string>Render Mode </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="renderModeComboBox">
        <item>
         <property name="text">
          <string>Single-threaded</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Multi-threaded CPU</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="threadCountLabel">
        <property name="text">
         <string>Thread count</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="threadCountSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="chunksSizeLabel">
        <property name="text">
         <string>Chunks size</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="chunksSizeSpinBox">
        <property name="minimum">
         <number>64</number>
        </property>
        <property name="maximum">
         <number>1024</number>
        </property>
        <property name="singleStep">
         <number>64</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="renderButton">
     <property name="text">
      <string>Render</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SquareSpinBox</class>
   <extends>QSpinBox</extends>
   <header>SquareSpinBox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
